app/src/lib/stores/notifications-store.ts

```diff
@@ -119,9 +129,90 @@ export class NotificationsStore {
         return this.handleChecksFailedEvent(e, skipNotification)
       case 'pr-review-submit':
         return this.handlePullRequestReviewSubmitEvent(e, skipNotification)
+      case 'pr-comment':
+        return this.handlePullRequestCommentEvent(e, skipNotification)
     }
   }
 
+  private async handlePullRequestCommentEvent(
+    event: IDesktopPullRequestCommentAliveEvent,
+    skipNotification: boolean
+  ) {
+    if (!enablePullRequestCommentNotifications()) {
+      return
+    }
+
+    const repository = this.repository
+    if (repository === null) {
+      return
+    }
+
+    if (!this.isValidRepositoryForEvent(repository, event)) {
+      if (this.isRecentRepositoryEvent(event)) {
+        this.statsStore.recordPullRequestReviewNotiificationFromRecentRepo()
+      } else {
+        this.statsStore.recordPullRequestReviewNotiificationFromNonRecentRepo()
+      }
+      return
+    }
+
+    const pullRequests = await this.pullRequestCoordinator.getAllPullRequests(
+      repository
+    )
+    const pullRequest = pullRequests.find(
+      pr => pr.pullRequestNumber === event.pull_request_number
+    )
+
+    // If the PR is not in cache, it probably means the user didn't work on it
+    // recently, so we don't want to show a notification.
+    if (pullRequest === undefined) {
```

Comment:
I was thinking -> would we get comments on closed PR's (those annoying after the fact spam ones is what came to mind). This answers that question; if it is closed, it will get removed from the PR cache.